packaging/amazon-linux-ami/ecs.service

```diff
@@ -15,7 +15,8 @@
 [Unit]
 Description=Amazon Elastic Container Service - container agent
 Documentation=https://aws.amazon.com/documentation/ecs/
-Requires=docker.service
+Requires=amazon-ecs-volume-plugin.service docker.service
+After=amazon-ecs-volume-plugin.service
 After=docker.service
```

Comment:
I'd leave this unchanged -- docker depends on the plugin, ecs only depends on docker.

Reply:
I added this so that the user does not have to start the volume plugin explicitly.

Reply:
synced offline with petderek@, since volume plugin will start on boot it does not need be started, so reverted the changes here.